examples/twitter-nlp/analyze_sentiment.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Script to analyze the sentiment of a given file of tweets from Twitter
+in batch processing mode.
+
+##########
+Command help:
+Usage: analyze_sentiment.py [OPTIONS]
+
+  Analyze the sentiment of the tweets given in the tweets_file and print out
+  the results.
+
+Options:
+  --model_path TEXT       The path to the sentiment analysis model to
+                          load.Either a model.onnx file, a model folder
+                          containing the model.onnx and supporting files, or a
+                          SparseZoo model stub.
+  --tweets_file TEXT      The path to the tweets json txt file to analyze
+                          sentiment for.
+  --batch_size INTEGER    The batch size to process the tweets with. A higher
+                          batch size may increase performance at the expense
+                          of memory resources and individual latency.
+  --total_tweets INTEGER  The total number of tweets to analyze from the
+                          tweets_file.Defaults to None which will run through
+                          all tweets contained in the file.
+  --help                  Show this message and exit.
+
+##########
+Example running a dense, unoptimized sentiment analysis model:
+python analyze_sentiment.py
+    --model_path "zoo:nlp/text_classification/bert-base/pytorch/huggingface/sst2/base-none"
+    --tweets_file /PATH/TO/OUTPUT/FROM/scrape.py
+"""
+
+import json
+from itertools import cycle, islice
+from typing import Dict, List, Optional
+
+import click
+from colorama import Fore
+
+try:
+    from deepsparse.transformers import pipeline
+except:
+    pass
```

Comment:
there is no fallback if pipeline doesn't exist. should we error then?

Reply:
this was a leftover from me testing locally on my mac, will remove